src/status_im/ui/screens/intro/views.cljs

```diff
@@ -119,39 +119,40 @@
                        :resize-mode :contain
                        :style {:width image-size :height image-size}}])])))
 
-(defn choose-key [{:keys [multiaccounts selected-id view-height]}]
+(defn choose-key [{:keys [multiaccounts selected-id]}]
   [react/view
    {:style {:flex            1
             :justify-content :center}}
    [react/view
-    {:style {:flex       1
-             :height     200
-             :max-height 338}}
+    {:style {:height 410}}
```

Comment:
are you sure this will work on small screens?

Reply:
Tested only on iPhone 8, will modify to support smaller screens.